stacker/hooks/aws_lambda.py

```diff
@@ -103,9 +103,11 @@ def _find_files(root, includes, excludes):
         root (str): base directory to list files from.
         includes (list[str]): inclusion patterns. Only files matching those
             patterns will be included in the result.
-        includes (list[str]): exclusion patterns. Files matching those
+        excludes (list[str]): exclusion patterns. Files matching those
             patterns will be excluded from the result. Exclusions take
             precedence over inclusions.
+        walk (bool): If true, symlinks will be included in the resulting
```

Comment:
any reason not to name this `follow_symlinks`?

Reply:
@GarisonLotus ^^ any reason not to name it follow_symlinks rather than walk?  Seems  more natural.